portfolio/src/main/webapp/index.html

```diff
@@ -206,6 +206,22 @@ <h5>Mount Diablo</h5>
       </div>
     </div>
 
+    <!-- Footer -->
+    <div id="footer-container">/
+        <a href="https://linkedin.com/in/ayakkala/">
+          <img class="logo-image" src="images/linkedIn-logo.png">
```

Comment:
Provide alternative contents for multimedia: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

Reply:
Ok, I will add that.